modules/run_protein_tools.py

```diff
@@ -0,0 +1,232 @@
+    arguments:
+        - seq (str): sequence for checking
+    return:
+        - bool: True or False depending on whether protein sequence is three-letter entry
+    """
+    seqset = {seq[i:i+3]  for i in range(0, len(seq), 3)}
+    return (seqset <= THREE_LETTER_ENTRY)
+    
+
+def is_seq_one_letter_protein(seq: str)->bool:
+    """
+    Check whether the protein sequence is one-letter entry
+    arguments:
+        - seq (str): sequence for checking
+    return:
+        - bool: True or False depending on whether protein sequence is one-letter entry
+    """
+    seqset = set(seq.upper())
+    return (seqset <= ONE_LETTER_ENTRY)
+
+
+def count_length(seq: str)->int:
+    """
+    Count the length of the protein sequence
+    arguments:
+        - seq (str): sequence for length counting
+    return:
+        - int: length of protein sequence  
+    """
+    if is_seq_one_letter_protein(seq):
+        return len(seq)
+    else:
+        return int(len(seq)/3)
+
+
+def count_percentage(seq: str)->dict:
+    """
+    Count percentage of each amino acid in sequence
+    arguments:
+        - seq (str): sequence for counting
+    return:
+        - dict: dictionary with counted percentage    
+    """
+    l = count_length(seq)
+    percentages = {}
+    if is_seq_one_letter_protein(seq):
+        step = 1
+        #print('step=', step)
+    else:
+        step = 3
+        #print('step=', step)
+    for i in range(0, len(seq), step):
+        aa = seq[i:i+step]
+        if aa not in percentages:
+            percentages[aa] = 1
+        else:
+            percentages[aa] += 1
+    percentages.update((key, round(value/l*100, 2)) for key, value in percentages.items())
+    percentages={key: value for key, value in sorted(percentages.items(), key=lambda item: item[1], reverse=True)}
+    return percentages
+
+
+def rename_another_letter_entry (seq: str)->str:
+    """
+    Transform to three-letter amino acids entry if initial is one-letter, and to one-letter if initial is three-letter
+    arguments:
+        - seqs (str): sequence for transforming to one(three)-letter entire
+    return:
+        - str: transformed sequence
+    """
+    renamed_seq = ''
+    if is_seq_one_letter_protein(seq):
+        step = 1
+        to_entry = TO_THREE_LETTER_DICT
+    else:
+        step = 3
+        to_entry = TO_ONE_LETTER_DICT
+    for i in range(0, len(seq), step):
+        aa = seq[i:i+step]
+        renamed_seq = renamed_seq + to_entry[aa]
+    return renamed_seq
+
+
+def transform_to_DNA_code(seq):
+    """
+    Transforming of an amino acid sequence/protein to DNA sequence
+    arguments:
+        - seqs (str): amino acid sequence of protein
+    return:
+        - str: sequence of protein in the DNA sequence form  
+    """
+    if is_seq_three_letter_protein(seq):
+        seq = rename_another_letter_entry(seq)
+    return ''.join([RETRANSLATE[aa] for aa in seq])
+
+
+def aa_property (seq: str)->str:

```

Comment:
```suggestion
def aa_property(seq: str) -> str:
```
Не совсем понятно из названия что же все таки делается.